sonar-dotnet-tests-library/src/main/java/org/sonar/plugins/dotnet/tests/XmlParserHelper.java

```diff
@@ -163,12 +162,12 @@ public void close() throws IOException {
       try {
         stream.close();
       } catch (XMLStreamException e) {
-        throw Throwables.propagate(e);
+        throw new RuntimeException(e);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00112)